### PR TITLE
Improved Documentation with Fixed Grammar Mistakes

### PR DESCRIPTION
**Pull request by SRKYousafzaiPK**
# Description

Improved documentation with fixed grammar mistakes to make documentation more easy to understand.

## Status
**READY**

## Type of change
- Documentation (fix or adds documentation)

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -38,12 +38,10 @@ skills:
...
+This tells opsdroid to use the built-in [shell connector](https://github.com/opsdroid/connector-shell) and [hello skill](https://github.com/opsdroid/skill-hello) from the official module library.
 
+In opsdroid modules can be git repositories which will be cloned locally the first time they are used. By default, if you do not specify a location for a module opsdroid will first look to see if it is built into the core and then look at https://github.com/opsdroid/<moduletype>-<modulename>.git for a repository. Therefore in the above configuration, the connector-shell module is found at opsdroid.connector.shell and the hello skill is found in the skill-hello repository from the opsdroid organization on GitHub.
+You are of course encouraged to write your modules and make them available on GitHub or any other repository host which is accessible by your opsdroid installation. We are especially keen for folks to contribute connectors and databases to the opsdroid core package.
```
Comment (jacobtomlinson) [inline docs/configuration.md]: 👎 Removing this line will make this harder to read.

File: docs/overview.md [hunk 2]
```diff
@@ -2,18 +2,18 @@
 
 ## ChatOps
 
-_"ChatOps is an operational paradigm where work that is already happening in the background today is brought into a common chatroom. By doing this, you are unifying the communication about what work should get done with actual history of the work being done."_ - [StackStorm](https://docs.stackstorm.com/chatops/chatops.html)
+_"ChatOps is an operational paradigm where work that is already happening in the background today is brought into a common chatroom. By doing this, you are unifying the communication about what work should get done with an actual history of the work being done."_ - [StackStorm](https://docs.stackstorm.com/chatops/chatops.html)
```
Comment (jacobtomlinson) [inline docs/overview.md]: 👎 This is a quote. You cannot change this text.

File: docs/overview.md [hunk 3]
```diff
@@ -2,18 +2,18 @@
 
 ## ChatOps
 
-_"ChatOps is an operational paradigm where work that is already happening in the background today is brought into a common chatroom. By doing this, you are unifying the communication about what work should get done with actual history of the work being done."_ - [StackStorm](https://docs.stackstorm.com/chatops/chatops.html)
+_"ChatOps is an operational paradigm where work that is already happening in the background today is brought into a common chatroom. By doing this, you are unifying the communication about what work should get done with a actual history of the work being done."_ - [StackStorm](https://docs.stackstorm.com/chatops/chatops.html)
```
Comment (jacobtomlinson) [inline docs/overview.md]: This is still a change. 

The original sentence was `By doing this, you are unifying the communication about what work should get done with a actual history of the work being done.`

**Comment by SRKYousafzaiPK** [comment]
Okay, thanks! 
I'll keep that in Mind :)

**Pull request merged**